src/lib/content/model/ContentItem.ts

```diff
@@ -16,7 +16,7 @@ export class ContentItem<T extends ContentBody = DefaultContentBody> {
    * Returns a plain JSON version of the content body, removing helper properties and functions.
    * This should be used if your application needs to serialize the content body, e.g. in an API response.
    */
-  toJSON() {
+  toJSON(): any {
```

Comment:
This is out of scope for this work but I think it would be really useful if we could define the return type of all the `toJson` functions.  Maybe allow a second generic type in addition to `T` which could extend a `ContentBodyJSON` interface (without all the helper functions etc).